CODEOWNERS

```diff
@@ -126,6 +126,8 @@
 /drivers/flash/*stm32*                    @superna9999
 /drivers/gpio/*ht16k33*                   @henrikbrixandersen
 /drivers/gpio/*stm32*                     @rsalveti @idlethread
+/drivers/gpio/Kconfig.litex               @mateusz-holenko @kgugala @pgielda
+/drivers/gpio/gpio_litex.c                @mateusz-holenko @kgugala @pgielda
```

Comment:
Can probably just make this `/drivers/gpio/*litex*`

Reply:
Changed.